app/state/reducers/searchFiltersReducer.js

```diff
@@ -0,0 +1,18 @@
+const initialState = {
+  search: '',
+};
+
+function searchFiltersReducer(state = initialState, action) {
+  switch (action.type) {
+    case 'ENTER_OR_CHANGE_SEARCH_PAGE': {
+      const filters = action.payload.query;
+      return Object.assign({}, initialState, filters);
```

Comment:
- The `{ ...initialState, ...filters }` syntax is in stage 3 now, so we could start using it. Not important to change here though.